server/models/image_item.py

```diff
@@ -90,6 +91,11 @@ def _createLargeImageJob(self, item, fileObj, user, token):
             user=user)
         jobToken = Job.createJobToken(job)
 
+        outputName = os.path.splitext(fileObj['name'])[0] + '.tiff'
+        if outputName == fileObj['name']:
+            outputName = (os.path.splitext(fileObj['name'])[0] + '.' +
```

Comment:
We could possibly just include the timestamp all the time as that would be generally informative as to when the conversion occurred (is that represented elsewhere in File model metadata?).